io/net/net_data.py

```diff
@@ -158,7 +158,7 @@ def testinterface(self):
             self.fail("interface test failed")
         # up the interface
         process.system(if_up, shell=True)
-        time.sleep(4)
+        time.sleep(5)
```

Comment:
The operation performed here is ifup of the interface and then a sleep is done for the change to be reflected. The interface state is checked after that and then determines PASS/FAIl.

it only fails because the sleep time is less and the state (even though it is up) is collected as fail